### ISSUE TITLE
inputTextarea not allowing spaces and carriage returns

### ISSUE THREAD

**Issue by mertsincan** [primefaces/primefaces#2015]
Reported By PRO User;
> Hello,
I have a paginated data table lazily loaded with dynamic columns where one of the columns is of type p:inputTextarea. I found that when I type text in the text area, all carriage returns and spaces typed by the user are not inserted in the text.
...

Test Code;
```
<p:dataTable id="carTable" var="car" value="#{dtLazyView.lazyModel}" paginator="true" rows="10" editable="true" 
                 paginatorTemplate="{CurrentPageReport}  {FirstPageLink} {PreviousPageLink} {PageLinks} {NextPageLink} {LastPageLink} Go to: {JumpToPageDropdown} Rows: {RowsPerPageDropdown} {Exporters}" paginatorPosition="top"
                 rowsPerPageTemplate="5,10,15" draggableColumns="true" editMode="row" selection="#{dtLazyView.selectedCar}" lazy="true" stickyHeader="true" reflow="true" rowHover="true" sortMode="single" >
              <p:column selectionMode="multiple" style="width:16px;text-align:center" exportable="false"/>
        <p:column headerText="Id" sortBy="#{car.id}" filterBy="#{car.id}">
                  <p:cellEditor>
                    <f:facet name="output">
                      <h:outputText value="#{car.id}" />
                    </f:facet>
                    <f:facet name="input">
                      <h:inputText value="#{car.id}" />
                    </f:facet>
                  </p:cellEditor>
        </p:column>
        <p:column headerText="Year" sortBy="#{car.year}" filterBy="#{car.year}">
                  <p:cellEditor>
                    <f:facet name="output">
                      <h:outputText value="#{car.year}" />
                    </f:facet>
                    <f:facet name="input">
                      <h:inputText value="#{car.year}" />
                    </f:facet>
                  </p:cellEditor>
        </p:column>
        <p:column headerText="Brand" sortBy="#{car.brand}" filterBy="#{car.brand}">
                  <p:cellEditor>
                    <f:facet name="output">
                      <h:outputText value="#{car.brand}" />
                    </f:facet>
                    <f:facet name="input">
                      <p:inputTextarea value="#{car.brand}" />
                    </f:facet>
                  </p:cellEditor>
        </p:column>
        <p:column headerText="Color" sortBy="#{car.color}" filterBy="#{car.color}">
                  <p:cellEditor>
                    <f:facet name="output">
                      <h:outputText value="#{car.color}" />
                    </f:facet>
                    <f:facet name="input">
                      <p:inputTextarea value="#{car.color}" />
                    </f:facet>
                  </p:cellEditor>
        </p:column>
              <p:column style="width:32px" exportable="false">
                <p:rowEditor />
              </p:column>
    </p:dataTable>
```

**Comment by the-killer-bee** [comment]
Hi mertsincan,

Do you know when primefaces 6.0.14 or 6.1 will be available as we have presentation next week and this issue is really awkward.

Thanks a lot

**Comment by kukel** [comment]
if the fix is in 6.1, you can most likely easily backport this to 6.0.13 (of which you do have the source)
